packages/vite/LICENSE.md

```diff
@@ -529,68 +443,50 @@ Repository: rollup/plugins
-> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-> copies of the Software, and to permit persons to whom the Software is
-> furnished to do so, subject to the following conditions:
-> 
-> The above copyright notice and this permission notice shall be included in
-> all copies or substantial portions of the Software.
-> 
-> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-> IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-> FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-> AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-> LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-> OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-> THE SOFTWARE.
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/bundle-name
 
----------------------------------------
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/default-browser
 
-## @rollup/pluginutils
 License: MIT
-By: Rich Harris
-Repository: rollup/plugins
+By: Sindre Sorhus
+Repository: sindresorhus/default-browser-id
 
-> The MIT License (MIT)
-> 
-> Copyright (c) 2019 RollupJS Plugin Contributors (https://github.com/rollup/plugins/graphs/contributors)
-> 
-> Permission is hereby granted, free of charge, to any person obtaining a copy
-> of this software and associated documentation files (the "Software"), to deal
-> in the Software without restriction, including without limitation the rights
-> to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-> copies of the Software, and to permit persons to whom the Software is
-> furnished to do so, subject to the following conditions:
-> 
-> The above copyright notice and this permission notice shall be included in
-> all copies or substantial portions of the Software.
-> 
-> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-> IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-> FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-> AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-> LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-> OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-> THE SOFTWARE.
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/define-lazy-prop
 
----------------------------------------
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/is-docker
 
-## ansi-regex
 License: MIT
 By: Sindre Sorhus
-Repository: chalk/ansi-regex
+Repository: sindresorhus/is-inside-container
+
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/is-wsl
+
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/open
+
+License: MIT
+By: Sindre Sorhus
+Repository: sindresorhus/run-applescript
+
+License: MIT
+By: Sindre Sorhus
+Repository: chalk/strip-ansi
```

Comment:
Would it be worth collapsing these lines but only do it when the By is the same in all repositories to keep things simple? It seems there are several cases like this
```
License: MIT
By: Sindre Sorhus
Repositories: ..., sindresorhus/run-applescript, chalk/strip-ansi
```

Reply:
Done 👍